hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicConfigXmlGenerator.java

```diff
@@ -401,6 +402,17 @@ public static void pnCounterXmlGenerator(ConfigXmlGenerator.XmlGenerator gen, Co
         }
     }
 
+    public static void externalDataStoreXmlGenerator(ConfigXmlGenerator.XmlGenerator gen, Config config) {
+        for (ExternalDataStoreConfig externalDataStoreConfig : config.getExternalDataStoreConfigs().values()) {
+            gen.open("external-data-store", "name", externalDataStoreConfig.getName())
```

Comment:
correct me if I'm wrong, but this will result in just multiple <external-data-store> nodes, maybe it's worth wrapping it into parent-level <external-data-stores>

Reply:
It was implemented according to the example from https://hazelcast.atlassian.net/browse/HZ-1144